src/gsy_e/models/strategy/state/storage_state.py

```diff
@@ -43,16 +49,28 @@ class ESSEnergyOrigin(Enum):
 EnergyOrigin = namedtuple("EnergyOrigin", ("origin", "value"))
 
 
+@dataclass
+class StorageLosses:
+    """Container for all loss related Storage settings."""
+
+    charging_loss_percent: float = 0.0
+    discharging_loss_percent: float = 0.0
+    self_discharge_per_day_kWh: float = 0.0
```

Comment:
All good, however the unit is striking as a bit odd here. Did you find it in any spec sheet of a battery? Reason I am asking is because I have mostly seen this unit being the percentage of the total capacity of the battery, e.g. https://www.batterypowertips.com/why-self-discharge-is-important-in-batteries/ or more specifically here https://www.batterypowertips.com/wp-content/uploads/2023/02/Why-is-self-discharge-important-in-batteries-Figure-1.jpeg .

Reply:
Oh yes, it is also a percentage value. I also found the links that you shared. Let me rename. Also, I need to fix the calculation then. I don't know why I came up with the current implementation.